step.yml

```diff
@@ -225,6 +230,12 @@ inputs:
     title: Additional options for the xcpretty command
     summary: Additional options to be added to the executed xcpretty command.
 
+- log_formatter_options:
```

Comment:
Similar to `xcpretty_options` above, wouldn't it be more consistent to call this `xcbeautify_options`?

Reply:
Renamed.